Khan/wonder-blocks
# WB-174: Update modal examples to use Button

Test Plan:
- load https://deploy-preview-224--wonder-blocks.netlify.com
- go to the Modal section and see that all buttons use the wonder-blocks Button component
- open the modals in the ModalLauncher section and see that they also use the wonder-blocks Button

![screen shot 2018-07-21 at 1 02 44 pm](https://user-images.githubusercontent.com/1044413/43038519-6690edfc-8ce8-11e8-9d5e-27d3810f51b6.png)
![screen shot 2018-07-21 at 1 08 23 pm](https://user-images.githubusercontent.com/1044413/43038522-6b659828-8ce8-11e8-8750-64af05d3f667.png)
![screen shot 2018-07-21 at 1 08 26 pm](https://user-images.githubusercontent.com/1044413/43038521-6b59d18c-8ce8-11e8-8b9b-972828078772.png)
